build-system/pr-check/validator-tests.js

```diff
@@ -37,6 +38,7 @@ const FILENAME = 'validator-tests.js';
 const FILELOGPREFIX = colors.bold(colors.yellow(`${FILENAME}:`));
 const timedExecOrDie = (cmd, unusedFileName) =>
   timedExecOrDieBase(cmd, FILENAME);
+const timedExec = (cmd, unusedFileName) => timedExecBase(cmd, FILENAME);
```

Comment:
Curious why the stylistic preference for `unusedFooBar` rather than just omitting it entirely

Reply:
Good catch. (I was merely using the existing convention). You're right that it's unnecessary, so I've cleaned up all the files.